### PR TITLE
Move the ks-devops helm chart into a new repo

### PR DESCRIPTION
**Pull request by LinuxSuRen**
Support to release ks-devops chart into breach [gh-pages](https://github.com/kubesphere/ks-devops/tree/gh-pages).

You can visit the helm chart via https://kubesphere-sigs.github.io/ks-devops-helm-chart/index.yaml once this PR got merged.

> Before that, you also can visit it from https://linuxsuren.github.io/ks-devops/index.yaml

Now you can see also https://github.com/kubesphere-sigs/ks-devops-helm-chart

I've tested it in https://github.com/kubesphere/ks-devops/issues/228

## Why not reuse?

Someone might know https://github.com/kubesphere/helm-charts . 

It's easy for us to maintain all the core parts of ks-devops in one repo. So, there is no good reason to move the source files of ks-devops chart into that repo.

/kind cleanup

### CODE REVIEW

**Comment by JohnNiang** [comment]
We need to update installation document as well. Please see:

https://github.com/LinuxSuRen/ks-devops/blob/cfb9d9574d7e5717b53b100cc2b14d1c385e2d73/docs/installation.md#install-via-helm

**Comment by LinuxSuRen** [comment]
> We need to update installation document as well. Please see:


Good catch

**Comment by JohnNiang** [comment]
/unhold 
LGTM

File: docs/installation.md [hunk 1]
```diff
@@ -8,10 +8,16 @@ kk create cluster --with-kubesphere nightly-$(date -d yesterday '+%Y%m%d')
 
 ## Install via helm
 
-First, please clone this git repository. Then run command: `make install-chart`
+First, add helm chart repo: `helm repo add ks https://kubesphere-sigs.github.io/ks-devops-helm-chart/`
```
Comment (JohnNiang) [inline docs/installation.md:11]: In this document, we name helm repo `ks`, but `ks-devops` is used in https://github.com/kubesphere-sigs/ks-devops-helm-chart#usage. That makes me confused. I think we should unify the names of two.

Comment (LinuxSuRen) [reply docs/installation.md:11]: So, what's your suggestion?

Comment (JohnNiang) [reply docs/installation.md:11]: I suggest using `ks-devops` as helm repo name. Please see https://openebs.github.io/charts/.

Comment (LinuxSuRen) [reply docs/installation.md:11]: Sure.

File: docs/installation.md [hunk 2]
```diff
@@ -8,10 +8,16 @@ kk create cluster --with-kubesphere nightly-$(date -d yesterday '+%Y%m%d')
...
 ## Install via helm
 
-First, please clone this git repository. Then run command: `make install-chart`
+First, add helm chart repo: `helm repo add ks-devops https://kubesphere-sigs.github.io/ks-devops-helm-chart/`
 
-> Because the default registry uses `ghcr.io/kubesphere`, if you want to use `docker.io` as the registry for image pull, 
-> you can exec `helm install ks-devops chart/ks-devops --set image.registry=kubespheredev`
+then, install it via:
+```shell
+helm install ks-devops ks/ks-devops -n kubesphere-devops-system --create-namespace \
```
Comment (JohnNiang) [inline docs/installation.md:15]: Please don't forget updating`ks/ks-devops`.

**Comment by JohnNiang** [comment]
/unhold
LGTM

**Pull request merged**